README.rst

```diff
@@ -40,19 +52,19 @@ Installation
 
     pip3 install -e git+https://github.com/flother/htmltab#egg=htmltab
 
-HTMLTab requires `Python 3`_, `Click`_, `lxml`_, cssselect_, and `Beautiful Soup 4`_.
+HTMLTab requires `Python 3`_, `Click`_, `lxml`_, cssselect_, `Requests`_ and `Beautiful Soup 4`_.
 
 Usage
 -----
 
 .. code-block:: text
 
-    Usage: htmltab [OPTIONS] EXPRESSION HTML_FILE
+    Usage: htmltab.py [OPTIONS] EXPRESSION
```

Comment:
The `.py` suffix isn't needed. If you install the library via pip or run `python setup.py develop`, [the `htmltab` program is made available] [1].

[1]: https://github.com/flother/htmltab/blob/cfa7d737634afc83072f0e82a2c580bb91467083/setup.py#L22